python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -516,18 +485,23 @@ async def enable_devices(self):
                 commands.CameraCableWrapPower(on=True),
                 # commands.CameraCableWrapEnableTracking(on=True),
             ]
-            await self.send_commands(*enable_commands)
+            await self.send_commands(*power_on_commands)
+        except salobj.ExpectedError as e:
+            self.log.error(f"Failed to power on one or more devices: {e}")
+            raise
         except Exception:
-            self.log.exception("Failed to enable one or more devices")
+            self.log.exception("Failed to power on one or more devices")
             raise
-        # if self.camera_cable_wrap_task.done():
-        #     self.camera_cable_wrap_task = asyncio.create_task(
-        #         self.camera_cable_wrap_loop()
-        #     )
+        self.camera_cable_wrap_follow_start_task = asyncio.create_task(
+            self.camera_cable_wrap_start_following()
+        )
+        await self.camera_cable_wrap_follow_start_task
 
     async def disable_devices(self):
         self.log.info("Disable devices")
         self.enable_task.cancel()
+        self.camera_cable_wrap_follow_start_task.cancel()
```

Comment:
Should you `await` for these tasks once they are cancelled?

```
try:
    await self.camera_cable_wrap_follow_start_task
except asyncio.CancelledError:
    pass
```

or something on those lines? Maybe with a timeout on the await?

Reply:
I don't see a need to wait, and not waiting makes the code simpler.